src/stan-users-guide/odes.Rmd

```diff
@@ -1,5 +1,54 @@
 # Ordinary Differential Equations  {#ode-solver.chapter}
 
+Stan provides three different built in methods for solving systems of
+ordinary differential equations (ODEs). If care is not taken, ODE models
+can be difficult to fit because
+
+- It is easy to make a mistake specifying the system of ODEs
+
+- A misspecified ODE can be very slow to solve -- making it difficult
+to quickly identify mistakes and interate
+
+- The numerical behavior of ODE can change radically in different parts
+of parameter space
+
+- There are a large number of different ODE solvers to pick from (how
+to identify the best settings for a problem is not trivial)
+
+- The sensitivity equations necessary to get gradients for HMC can make
+the ODE system scaling seem unusually expensive
+
+These problems are only amplified by the reality of Bayesian workflow,
```

Comment:
Is it "Bayesian workflow" or "the Bayesian workflow"? I know we use the former often but it bugs me a little.